### PR TITLE
feat: removing plausible provider from project

### PR DESCRIPTION
**Pull request by DhairyaMajmudar**
**What kind of change does this PR introduce?**

Removing Plausible provider from the project.  

**Issue Number:**
N/A

**Screenshots/videos:**
N/A

**If relevant, did you update the documentation?**
N/A

**Summary**
Plauslibe provider removed.

**Does this PR introduce a breaking change?**
N/A

### CODE REVIEW

File: components/Layout.tsx
Comment (Relequestual) [inline components/Layout.tsx:1]: Only need to remove the Plausible Provider. We need to keep the script though.

The issue was the script tag and the Plausible Provider together were loading the script twice, and we only need it once.

**Review by Relequestual — CHANGES_REQUESTED** [review]

File: components/Layout.tsx
Comment (benjagm) [reply components/Layout.tsx:1]: Exactly. We need to leave Layout.tsx intact.

**Review by benjagm — CHANGES_REQUESTED** [review]
Great job removing PlausibleProvider but we need to leave Layout.tsx intact with the official script.

File: components/Layout.tsx
Comment (DhairyaMajmudar) [reply components/Layout.tsx:1]: Sure thing, I have reverted the changes from the `Layout.tsx` file : )

**Review by benjagm — APPROVED** [review]
Thanks Dhairya!

**Pull request merged**